### ISSUE TITLE
Remove vue

### ISSUE THREAD

**Issue by Ashwagandhae** [lukajaa/bay-clock-2#8]
Dear Mr. "Lookazha",

I am writing to you with a deep and burning passion in my heart for the Bay Clock 2 app. As a web developer who has devoted my life to creating applications that are simple, efficient, and accessible to all, I am outraged and deeply disappointed by your decision to use Vue.

Let me make one thing clear: Vue is a fantastic tool for building complex and dynamic web applications. But when it comes to a simple app like Bay Clock 2, Vue is nothing but an unwarranted obstacle to our ultimate goal. Vue is a framework, not just a library, and it introduces an avalanche of overhead and complexity that is not only unnecessary but downright harmful to the app's performance and maintainability.

Not only that, but using Vue also means that the app is tied to a specific technology, which makes it less portable and adaptable in the long run. We cannot allow ourselves to be held hostage by the fickle whims of technology. By using pure HTML and JavaScript, we ensure that the app can be easily maintained and updated, regardless of the technological landscape.

And let's not forget about the accessibility issue. As developers, we have a moral obligation to ensure that the apps we create are accessible to as many people as possible. Using Vue, with its heavy reliance on JavaScript, can make the app slow and inaccessible for users with slower devices or internet connections. This is unacceptable, and it fills me with an unbridled rage.

We cannot continue down this path of over-engineering and complexity. We must take a stand for simplicity, efficiency, and accessibility. We must build apps that are easy to maintain, adaptable, and accessible to all. We must reject the status quo and blaze a new trail towards a brighter future.

In conclusion, I urge you to reconsider your decision to use Vue for Bay Clock 2. Let's use pure HTML and JavaScript instead and make the app more efficient and accessible to all. Let's unleash a storm of passion and take the world by storm! Let's create a future where simplicity and efficiency reign supreme!

**Comment by lucaskchang** [comment]
Dear Mr. "Ash-wag-and-hay",

As an experienced web developer, I too think that these issues are very important. Let me take some time to address each of your concerns. Additionally, as a man of science, I feel as though we should draw from real data to draw our conclusions. I've divided your complaints into two main sections: developer and client use. I will analyze each, one at a time, to provide a clear view of why I chose to use the Vue framework.

**Developer Use**
I initially chose to use Vue because it was the easiest framework to learn. The ability for 6th grade me to use Vue is indicative of its accessibility for all. Combining the stats of my original repo and this v2 one, I have received 5 total forks of the repo. Given that the number of people at my school that can code is meager to being with, the amount of people that have been able to build off of my code is astounding. 

This can probably be attributed to the excellent README situated in my repo. It lays clearly and easily how changing the simple functionality of Bay Clock can be done.

Outside of my community, the Vue github repository has received over 202,000 stars. Even the more specific Vue framework I am using, Quasar, has received over 22,000 stars. If needed, there will be always be a willing and helpful community of programmers to help whoever needs help with using my program.

**Client Use**
You mentioned that using Vue "can make the app slow and inaccessible for users with slower devices or internet connections." However, this is not a problem due to the tree-shaking in the frameworks I am using, allowing for a quick and lightweight app. Without images, the distribution branch of Bay Clock 2 weighs in at a light 990,223 bytes. I can exclude images because the nature of Vue's conditional rendering means that images aren't loaded, unless a user specifically does an action that requires them to be.

In fact, since the release of the newest version of Bay Clock, I have received multiple adulations regarding how quickly the site loads.

**Hypocrisy**
You mention Vue may be a bit overkill for an app like Bay Clock. However, with over 300 users a day, it is necessary to use frameworks that reflect the cutting edge of technology. 

If I may, I will compare this to your [framework-speak](https://github.com/Ashwagandhae/framework-speak) app.  Your distribution branch weighs in at 1,079,937 bytes, almost 90,000 bytes larger than my app. Your app receives barely one percent of my daily users (if we're being generous), yet somehow had the audacity to use almost 100 kilobytes more than me. 
<img width="264" alt="Screenshot 2023-02-24 at 10 10 39 AM" src="https://user-images.githubusercontent.com/39248525/221256938-1e053387-ebef-4748-886e-0dc740cf90f4.png">
<img width="268" alt="Screenshot 2023-02-24 at 10 11 03 AM" src="https://user-images.githubusercontent.com/39248525/221257013-7eda5031-d82c-414c-afbb-7351706507f9.png">

Furthermore, the repositories of Rust and Svelte have less stars than Vue even when they are combined. So, really, who is making esoteric, hard to use, and unnecessarily heavy web apps?

Best, 
Luca Chang
